deploy/sample-backup-configs/sample_config_build_recipes.ini

```diff
@@ -5,12 +5,12 @@
 # edit config_build.ini to actually "turn on" a config to be built when you run
 # buildafi
 
-#[firesim-singlecore-nic-lbp]
-#DESIGN=FireSim
-#TARGET_CONFIG=FireSimRocketChipSingleCoreConfig
-#PLATFORM_CONFIG=FireSimConfig
-#instancetype=c4.4xlarge
-#deploytriplet=None
+[firesim-singlecore-nic-lbp]
```

Comment:
Are we changing our default provided AFIs?

Reply:
I uncommented this to do some testing; i can put it back in